src/vuestic-theme/vuestic-components/va-input/VaInput.vue

```diff
@@ -145,6 +174,12 @@ export default {
         }
       )
     },
+    hasContent () {
+      return this.value != null && this.value !== ''
```

Comment:
Curious why not just `!this.value`?

Reply:
actually I'm not sure if an input would only emit/accept string values (maybe type number/boolean?) We can use `!value` if we decided that `false` or `0` are still consider an empty value.

I happened to add that check just because I used to pass numbers (instead of strings) into input[type=number] , and is frustrated when passing a default value `0` will make the input thinks there is no value.

Feel free to correct me :)

Reply:
I changed to `[null, undefined, ''].includes(this.value)`. Makes a bit more sense to me,